android/hybridcommon/build.gradle.kts

```diff
@@ -0,0 +1,85 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
+    alias(libs.plugins.androidJunit5)
+    alias(libs.plugins.mavenPublish)
+    alias(libs.plugins.detekt)
+}
+
+dependencies {
+    detektPlugins(libs.detekt.formatting)
+}
+
+detekt {
+    buildUponDefaultConfig = true
+    baseline = file("config/detekt/detekt-baseline.xml")
+}
+
+android {
+    namespace = "com.revenuecat.purchases.hybridcommon"
+    compileSdk = 34
+
+    defaultConfig {
+        // TODO: revert this to 19
+        minSdk = 24
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
+dependencies {
+
+    implementation(libs.core.ktx)
```

Comment:
Do these include annotation and fragment? Because we used to have 

```
    implementation 'androidx.annotation:annotation:1.3.0'
    implementation 'androidx.fragment:fragment-ktx:1.6.2'
    ```

Reply:
I believe so, at least, it compiles and runs tests fine.